### ISSUE TITLE
(question) Haffine_from_points , image_in_image

### ISSUE THREAD

**Issue by zzangho** [jesolem/PCV#10]
Hi, I'm student studying your book
thank you for your good teaching

I have one question ( book page 56,58 , chapter 3)

In the definition of Haffine_from_points(fp,tp),
the first argument is 'fp' and the second argument is 'tp'

but in the definition of image_in_image,
you switched the argument of Haffine_from_points function
in this case, first argument is 'tp' and second argument is 'fp'

i can't match definition and application of Haffine_from_points

Why did you do that?

thank you for your reading, if you don't understand, i'll rewrite

**Comment by karesh** [comment]
yes I have the same question. beatles image should transform/warp to the billboard_for_rent image. Why you switched the points. ?
